staging/src/k8s.io/kubectl/pkg/explain/v2/template.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"embed"
+	"path/filepath"
+	"strings"
+)
+
+//go:embed templates
```

Comment:
I don't know what this does, can you explain? I guess this loads the files into the binary, thanks.

Reply:
go:embed causes the files in the `templates` folder to be baked into the binary. The comment populates the below declaration with the file contents so they may be read